tests/buildtest/crates/invalid-cargotoml-missing-override/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "invalid-cargotoml-missing-override"
+version = "0.1.0"
+authors = ["Pietro Albini <pietro@pietroalbini.org>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+build-rs = "=0.1.2"
+
+[replace]
```

Comment:
can you explain what's missing here?

Reply:
A reproduction of [this error](https://crater-reports.s3.amazonaws.com/beta-1.86-1/1.85.0/gh/bestform.game-of-life.rs/log.txt) error if I am not mistaken